### PR TITLE
Fix disturbance documents

### PR DESCRIPTION
**Pull request by 200km**
## Overview
Fix disturbance documents to suit with the v6.0.0 source codes.

## Issue
https://github.com/ut-issl/s2e-core/pull/303

## Details
- The file names, class names described in the documents were modified.
- Old descriptions were removed.
- The file names of documents for gravity gradient and magnetic disturbances were changed.

##  Validation results
NA

## Scope of influence
NA

## Supplement
NA

## Note
NA

### CODE REVIEW

File: Specifications/Disturbance/Spec_Disturbance.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+1. Functions
+   - `Disturbance` is a base class for all disturbance classes.
+
+2. Related files
+   - `src/disturbances/disturbance.hpp`
+
+3. How to use   
+   - When you want to make a new disturbance class, you need to inherit this.
+   - Users need to define the `Update` function according to the target disturbance calculation. And update the members `force_b_N`, `torque_b_Nm`, `acceleration_b_m_s2_`, and/or `acceleration_i_m_s2_` in the function.
```
Comment (suzuki-toshihir0) [inline Specifications/Disturbance/Spec_Disturbance.md:13]:
```suggestion
   - Users need to define the `Update` function according to the target disturbance calculation. And update the members `force_b_N_`, `torque_b_Nm_`, `acceleration_b_m_s2_`, and/or `acceleration_i_m_s2_` in the function.
```

[NITS] （細かいですが）変数名を修正

File: Specifications/Disturbance/Spec_GeoPotential.md [hunk 2]
```diff
@@ -6,19 +6,21 @@
...
 3. How to use   
-   - Make an instance of the geopotential class in `InitializeInstances` function in `Disturbances.cpp`
-   - Chose RK4 orbit propagator
-   - Select `ENABLE` and `degree` of calculation
-     - When the `degree` is smaller than 1, it is overwritten as 0.
-     - When the `degree` is larger than 360, it is overwritten as 360.
-     - **NOTE**: The calculation time is related to the selected degree.
+   - Make an instance of the `GeoPotential` class in `InitializeInstances` function in `disturbances.cpp`
+     - Create an instance by using the initialization function `InitGeoPotential`
+   - Chose a orbit propagator mode that considers disturbances.
```
Comment (suzuki-toshihir0) [inline Specifications/Disturbance/Spec_GeoPotential.md:17]:
```suggestion
   - Choose an orbit propagator mode that considers disturbances.
```

[NITS] typo

**Review by suzuki-toshihir0 — APPROVED** [review]
細かい点だけコメントしていますが，対応後マージしていただいて問題ありません！

File: Specifications/Disturbance/Spec_GeoPotential.md (hunk 2 above)
Comment (200km) [reply Specifications/Disturbance/Spec_GeoPotential.md]: ありがとうございます。修正したのでマージします。

**Pull request merged**